packages/framework/presence/src/presenceManager.ts

```diff
@@ -30,16 +30,26 @@ import type {
 import { createEmitter } from "@fluid-experimental/presence/internal/events";
 
 /**
+ * Requirements for components connecting inside (under) the presence manager.
+ *
  * @internal
  */
-export interface IPresenceManager
-	extends IPresence,
-		Pick<Required<IContainerExtension<[]>>, "processSignal"> {}
+export type PresenceInternal = Pick<IPresence, "getAttendee">;
```

Comment:
Clarification mostly for me as I needed more context - what does "internal" boundary here represent? Again, its always better when name represents intended function/purpose rather then scope of usage.